server/services/adjustmentsService.ts

```diff
@@ -14,10 +14,6 @@ export default class AdjustmentsService {
     return new AdjustmentsClient(token).findByPerson(person)
   }
 
```

Comment:
This `findByPersonAndSource` function isnt being used by anything so removed to avoid having to maintain it